platform/solr/solr-factory/src/main/java/org/codice/solr/factory/SolrServerFactory.java

```diff
@@ -164,7 +164,7 @@ static SolrServer getHttpSolrServer() {
         SolrServer server;
         try {
             server = getSolrServer(url, coreName, configFile, coreUrl);
-        } catch (SolrException ex) {
+        } catch (Exception ex) {
```

Comment:
This seems a little broad. Do we really want this `catch` block to handle exceptions like `NullPointerException` as well?